grid-client/node/node.go

```diff
@@ -374,8 +374,15 @@ func (n *NodeClient) SystemVersion(ctx context.Context) (ver Version, err error)
 	return
 }
 
+// TaskResult holds the perf test result
+type TaskResult struct {
+	Name      string      `json:"name"`
+	Timestamp uint64      `json:"timestamp"`
```

Comment:
there is a description field missing here, maybe we should imported from zos directly so we don't have to maintain two identical structs

Reply:
nvm we will not import the type :)
adding the missing field is enough